src/main/java/com/mcmoddev/lib/integration/plugins/TinkersConstructBase.java

```diff
@@ -1,15 +1,12 @@
 package com.mcmoddev.lib.integration.plugins;
 
 import javax.annotation.Nonnull;
-
 import com.mcmoddev.lib.init.Materials;
 import com.mcmoddev.lib.integration.IIntegration;
 import com.mcmoddev.lib.integration.plugins.tinkers.ModifierRegistry;
 import com.mcmoddev.lib.integration.plugins.tinkers.TCMaterial;
 import com.mcmoddev.lib.integration.plugins.tinkers.TinkersConstructRegistry;
 import com.mcmoddev.lib.material.MMDMaterial;
-import com.mcmoddev.lib.util.ConfigBase.Options;

```

Comment:
MMDLib is likely going to need a config handler - with the TODO I've seen you putting together, can you start adding some things, like the need to add a config system to MMDLib itself and not just the base, but actual config for the lib (like, controls on its provided plugins, etc...)

Reply:
sure thing... but will probably not be part of this PR... :)